requirements_examples_genai.txt

```diff
@@ -13,12 +13,13 @@
 # limitations under the License.
 # ==============================================================================
 # LINT.IfChange
-gunicorn>=20.1.0
+-r requirements_examples_common.txt
+
+google-cloud-aiplatform>=1.60.0
+keras>=3.0.0
+keras-nlp>=0.14.0
 sentencepiece==0.1.99
-tensorflow>=2.10.0,<2.16.0
-tensorflow-datasets>=4.9.0
-tensorflow-text>=2.10.0,<2.16.0
+tensorflow-text>=2.16.0
```

Comment:
I believed tydi require tensoflor-text < 2.16.0.  Check change list 646547287.

Reply:
TyDI is still separated from the main demos and unaffected by this change.